mmv1/third_party/terraform/services/compute/resource_compute_instance_from_template.go.erb

```diff
@@ -262,6 +262,15 @@ func adjustInstanceFromTemplateDisks(d *schema.ResourceData, config *transport_t
 						// only have the name (since they're global).
 						disk.InitializeParams.DiskType = fmt.Sprintf("zones/%s/diskTypes/%s", zone.Name, dt)
 					}
+					if rp := disk.InitializeParams.ResourcePolicies; len(rp) > 0 {
```

Comment:
do we have a test for this logic?

Reply:
currently working on acctests for google_compute_instance_from_template and google_compute_instance_from_machine_image that were requested by the modular-magician. It'll be tested then